src/service-worker.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
Both service-worker are for using the app as a PWA, I am almost done, just have to read up so more on the manifest, since it´s not completely working in lighthouse.